File: L2/C/sala/1.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
// 
int main(){
    float n=0.0;
    char*inp=(char*)malloc(2*sizeof(inp)),*split;
    printf("Insira as notas do aluno(uma linha): ");
    fgets(inp,10,stdin);
    for(int i=0;split=strtok_r(inp," ",&inp);++i){
        n+=atof(split)/3.0;
    }
    if(n<6.0){
        printf("Reprovado(ao som do seu Madruga)! ");
    } else{
        printf("Aprovado! ");
    }
    printf("Independente da sua condicao, voce tirou %.2f",n);
    return 0;
}

File: L2/PY/sala/1.py
x=list(map(float,input('Insira as notas do aluno: ').split()))
v=0
for i in x:
    v+=i/len(x)
print("Reprovado",end='. ') if v<6 else print('Aprovado',end='. ')
print('Independentemente, a sua nota foi {:.2f}'.format(v))